skriptas.js

```diff
@@ -0,0 +1,42 @@
+const slider = document.querySelector("#image-comparison-slider");
+const sliderImgWrapper = document.querySelector("#image-comparison-slider .img-wrapper");
+const sliderHandle = document.querySelector("#image-comparison-slider .handle");
+
+slider.addEventListener("mousemove", sliderMouseMove);
+slider.addEventListener("touchmove", sliderMouseMove);
+
+function sliderMouseMove(event){
+
+    if (isSliderLocked) return;
+
+    const sliderLeftX = slider.offsetLeft;
+    const sliderWidth = slider.clientWidth;
+    const sliderHandleWidth = sliderHandle.clientWidth;
+
+    let mouseX = (event.clientX || event.touches[0].clientX) - sliderLeftX;
+    if (mouseX < 0) mouseX =0;
+    else if (mouseX > sliderWidth) mouseX = sliderWidth;
```

Comment:
Kažkas neidealiai veikia jei pelę staigiai perbrauki, lieka tarpas. Galima būtų bandyt įdėt buferinę zoną aplik sliderį, kad eventai dar trigerintūsi šiek tiek į kairę ir į dešinę.
![image](https://user-images.githubusercontent.com/36815064/189962618-e8c0ad5b-cb7f-458b-8e15-a9b4e75ab884.png)